.circleci/continue_config.yml

```diff
@@ -0,0 +1,771 @@
+# version: 2.1 # use CircleCI 2.1
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2.1
+
+orbs:
+  slack: circleci/slack@4.1.3
+
+# the default pipeline parameters, which will be updated according to
+# the results of the path-filtering orb
+parameters:
+  global-change:
+    type: boolean
+    default: false
```

Comment:
Set `global-change` to false by default.

Unsure how this will work with master. Will have to test once merged, but may have to revert worst case.

Reply:
Nit: Would it make sense to call it `global_change` (snake case) for consistency?